src/cmds/build/bundle.ts

```diff
@@ -1,37 +1,36 @@
-const { promises: fs, createWriteStream } = require('fs');
-const browserify = require('browserify');
-const stripComments = require('strip-comments');
-// const terser = require('terser')
+import { promises as filesystem, createWriteStream } from 'fs';
+import { logError } from '../../utils';
+import browserify from 'browserify';
+import stripComments from 'strip-comments';
```

Comment:
I'm surprised eslint doesn't complain about this, but the order of imported packages should go like this:

1. Native Node dependencies (like `fs`)
2. Third-party dependencies
3. Files in this package that belong to a parent directory or different children of a parent directory (i.e. any path starting with `../`)
4. Files in the same directory as this file

Reply:
I wrote the above comment before I realized we're not linting the TypeScript files 😛